tiatoolbox/models/architecture/utils.py

```diff
@@ -0,0 +1,93 @@
+# ***** BEGIN GPL LICENSE BLOCK *****
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# The Original Code is Copyright (C) 2021, TIALab, University of Warwick
+# All rights reserved.
+# ***** END GPL LICENSE BLOCK *****
+
+"""Defines utlity layers and operators for models in tiatoolbox."""
+
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+
+def crop_op(x, cropping, data_format="NCHW"):
```

Comment:
Maybe 'centre_crop' would be more descriptive as it lets the user know that is a centred crop and not to expect to give arguments for location.